[2024-07-17T08:10:45.781+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Webscraping.Pre-Scrape_script manual__2024-07-17T08:05:42.016473+00:00 [queued]>
[2024-07-17T08:10:45.790+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Webscraping.Pre-Scrape_script manual__2024-07-17T08:05:42.016473+00:00 [queued]>
[2024-07-17T08:10:45.790+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 2
[2024-07-17T08:10:45.804+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): Pre-Scrape_script> on 2024-07-17 08:05:42.016473+00:00
[2024-07-17T08:10:45.809+0000] {standard_task_runner.py:60} INFO - Started process 83 to run task
[2024-07-17T08:10:45.812+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Webscraping', 'Pre-Scrape_script', 'manual__2024-07-17T08:05:42.016473+00:00', '--job-id', '138', '--raw', '--subdir', 'DAGS_FOLDER/webscrape_dag.py', '--cfg-path', '/tmp/tmptn10hide']
[2024-07-17T08:10:45.813+0000] {standard_task_runner.py:88} INFO - Job 138: Subtask Pre-Scrape_script
[2024-07-17T08:10:45.868+0000] {task_command.py:423} INFO - Running <TaskInstance: Webscraping.Pre-Scrape_script manual__2024-07-17T08:05:42.016473+00:00 [running]> on host 56f40d5e3a97
[2024-07-17T08:10:45.957+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Webscraping' AIRFLOW_CTX_TASK_ID='Pre-Scrape_script' AIRFLOW_CTX_EXECUTION_DATE='2024-07-17T08:05:42.016473+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-17T08:05:42.016473+00:00'
[2024-07-17T08:10:45.959+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-17T08:10:45.959+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /webscrape_scripts/pre_scrape.py']
[2024-07-17T08:10:45.968+0000] {subprocess.py:86} INFO - Output:
[2024-07-17T08:10:45.991+0000] {subprocess.py:93} INFO - python3: can't open file '/webscrape_scripts/pre_scrape.py': [Errno 2] No such file or directory
[2024-07-17T08:10:45.993+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2024-07-17T08:10:46.005+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2024-07-17T08:10:46.007+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=Webscraping, task_id=Pre-Scrape_script, execution_date=20240717T080542, start_date=20240717T081045, end_date=20240717T081046
[2024-07-17T08:10:46.023+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 138 for task Pre-Scrape_script (Bash command failed. The command returned a non-zero exit code 2.; 83)
[2024-07-17T08:10:46.067+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-07-17T08:10:46.090+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
